.gitlab-ci.yml

```diff
@@ -19,6 +19,21 @@ cppcheck:
   tags:
     - docker
 
+flake8:
+  stage: build
+  image: pipelinecomponents/flake8:latest
```

Comment:
I'm personally not a big fan of using "rolling" images like "latest", because it makes it harder to reproduce tests if the image was silently updated from one job to another.
On the other hand, it avoids the need to manually check and update images and it's frequently used, so I guess it's a split.